File: pset6/sentimental-mario-more/mario.py
while True:
    try:
        height = int(input("Height: "))
        if height >= 1 and height <= 8:
            break
    except ValueError:
        pass

for row in range(1, height + 1):
    for space in range(height - row):
        print(" ",end='')

    for left in range(1, row + 1):
        print("#",end='')

    print("  ",end='')

    for right in range(1, row + 1):
        print("#",end='')
    print()

File: pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>


int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);


    for (int row = 1; row <= height; row++)
    {

        for (int space = 0; space < height - row; space++)
        {
            printf(" ");
        }


        for (int left = 0; left < row; left++)
        {
            printf("#");
        }


        printf("  ");


        for (int right = 0; right < row; right++)
        {
            printf("#");
        }

        
        printf("\n");
    }
}
